### PR TITLE
M5G-147 - MessagingBubble 

### PR DESCRIPTION
**Pull request by jonjonray**
**Jira:**
[M5G-147](https://clever.atlassian.net/browse/M5G-147?atlOrigin=eyJpIjoiMzIxODJkMjllN2MxNGIwMTkxYWU2NWQ4MDA4ZTRkMjEiLCJwIjoiaiJ9)

**Overview:**
Adding a `replyTo` prop for the messagingBubble component to support replies to announcements. 

**Screenshots/GIFs:**

Screen Shot 2021-04-05 at 4 57 18 PM

Screen Shot 2021-04-05 at 4 34 25 PM

Original Designs

Screen Shot 2021-04-05 at 5 14 08 PM

Screen Shot 2021-04-05 at 5 13 56 PM

**Testing:**
Tested locally

- [x] Unit tests
- Manual tests:
  - [x] Chrome
  - [x] Safari
  - [ ] IE11

**Roll Out:**
100
- Before merging:
  - [x] Updated docs
  - [x] Bumped version in `package.json`
    - Breaking change?
      - If it is a beta component run `npm version minor`
      - If the component is not in beta run `npm version major`
    - New component or backward-compatible component feature change? Run `npm version minor`
    - Only changing documentation? All good. Skip this step.
  - After creating a new component, make sure to add it to the Components List in `ComponentsView.jsx`. To do so:
  -
- After merging:
  - [TODO] Deployed updated docs (`make deploy-docs`)

### CODE REVIEW

File: src/MessagingBubble/MessagingBubble.less [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
-  color: @neutral_white;
   font-size: 1rem;
   max-width: 25rem;
-  overflow-wrap: break-word;
   .padding--xs();
   .text--line-height-4();
+  overflow-wrap: break-word;
   white-space: pre-wrap;
   word-wrap: break-word; // IE name for overflow-wrap
 }
```
Comment (jonah-clever) [inline src/MessagingBubble/MessagingBubble.less:13]: nit: add a newline here?

File: src/MessagingBubble/MessagingBubble.less [hunk 2]
```diff
@@ -30,7 +30,7 @@
...
 
 .MessagingBubble--Message--Reply--Other {
-  background-color: #fafbfe; /* stylelint-disable-line */
+  background-color: @neutral_off_white;
```
Comment (jonah-clever) [inline src/MessagingBubble/MessagingBubble.less:34]: nice

**Review by jonah-clever — APPROVED** [review]
🌮

File: src/MessagingBubble/MessagingBubble.less (hunk 1 above)
Comment (jonjonray) [reply src/MessagingBubble/MessagingBubble.less:13]: tried adding a new line and the linter is not a huge fan

**Pull request merged**